pkg/mock/fileUtils.go

```diff
@@ -378,7 +388,7 @@ func (f *FilesMock) TempDir(baseDir string, pattern string) (string, error) {
 		tmpDir = fmt.Sprintf("%s/%stest", baseDir, pattern)
 	}
 
-	err := f.MkdirAll(tmpDir, 0755)
+	err := f.MkdirAll(tmpDir, 0o755)
```

Comment:
You could use the constants here and elsewhere.

Reply:
I actually did not really touched this. This is automatically done by some plugin ...
I did not want to touch even more aspects in this PR ;-)